drivers/platform/maxim/maxim_uart.c

```diff
@@ -265,30 +265,30 @@ int32_t no_os_uart_init(struct no_os_uart_desc **desc,
 		parity = MXC_UART_PARITY_DISABLE;
 		break;
 	case NO_OS_UART_PAR_MARK:
-#if TARGET_NUM == 32660 || TARGET_NUM == 32650
+#if TARGET_NUM == 32660 || TARGET_NUM == 32650 || TARGET_NUM == 32665
```

Comment:
i'm wondering if there is an alternative to these ifdefs (we keep expanding them, and it starts to look a bit ugly :) )